docs/getting-started/integrations-cdn/react-default-cdn.md

```diff
@@ -38,7 +38,228 @@ You get ready-to-use code tailored to your needs!
 
 This guide assumes you have a React project. You can create a basic React project using [Vite](https://vitejs.dev/). Refer to the [React documentation](https://react.dev/learn/start-a-new-react-project) to learn how to set up a project in the framework.
 
-## Component properties
+### Installing react component from npm
+
+Install the `@ckeditor/ckeditor5-react` package:
+
+```bash
+npm install @ckeditor/ckeditor5-react
+```
+
+The `useCKEditorCloud` hook is responsible for returning information that:
+
+- CKEditor is still being downloaded from the CDN with status = "loading".
+- An error occurred during the download when status = "error", then further information is returned in the error field.
+- Returning the editor in the data field and its dependencies when status = "success".
+
+Use the `<CKEditor>` component inside your project. The below example shows how to use the component with the open-source plugins.
+
+```js
+import React from "react";
+import { CKEditor, useCKEditorCloud } from "@ckeditor/ckeditor5-react";
+
+const CKEditorDemo = () => {
+	const cloud = useCKEditorCloud( {
+		version: "43.0.0",
```

Comment:
Maybe better to use current release version, WDYT?

```suggestion
		version: "{@var ckeditor5-version}",
```

Reply:
I'm all in for that solution, we won't need to update it manually.

Reply:
I wondered if the variable would work in snippets. And yes - it does. So, it's a good idea.